models/eprop_readout.cpp

```diff
@@ -330,9 +370,15 @@ eprop_readout::compute_gradient( const long t_spike,
 
   for ( long t = t_spike_previous; t < t_compute_until; ++t, ++eprop_hist_it )
   {
-    z = z_previous_buffer;
-    z_previous_buffer = z_current_buffer;
-    z_current_buffer = 0.0;
+    if ( P_.delay_rec_out_ > 1 )
+    {
+      z = z_previous_buffer.front(); // <<-------------------------
+      update_pre_syn_buffer_multiple_entries( z, z_current, z_previous, z_previous_buffer, t_spike, t );
```

Comment:
Is the `z_previous_buffer.front()` correct here?
```suggestion
      z = z_previous_buffer.front();
      update_pre_syn_buffer_multiple_entries( z, z_current, z_previous, z_previous_buffer, t_spike, t );
```